remotes/docker/authorizer.go

```diff
@@ -36,13 +35,15 @@ import (
 	"golang.org/x/net/context/ctxhttp"
 )
 
+const defaultExpiredIn = 60
```

Comment:
Ah, oops, I missed the mis-spell; note that I didn't do a real review, I just noticed this PR and spotted the `var/const`, and now the typo; I'm not super-familiar with this area of the code, so you might want to wait for "real" reviewers to do their review 😅

```suggestion
const defaultExpiresIn = 60
```